domains/mercadoLibre/index.js

```diff
@@ -251,6 +255,15 @@ class MercadoLibreDomain {
     })
     return response
   }
+
+  async updateActive(id, bodyData, options = {}) {
+    const { transaction = null } = options
+    const active = pathOr(null, ['active'], bodyData)
+
+    const adUpdated = await MlAdModel.findByPk(id)
+
+    return await adUpdated.update(active, { transaction })
+  }
```

Comment:
pra esse função funcionar da forma que deve o bodyData deve se 
```
{
  active: { active: true/false } 
}
```
se ele for enviado como: 
```
{
  active: true/false 
}
```
a função adUpdated.update receberá como primeiro parametro apenas um boleano, e ela espera um objeto contendo as chave-valor a serem atualizados